### PR TITLE
Update CHANGES.md for the edge-18.12.1 release

### PR DESCRIPTION
**Pull request by rmars**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+  * **New** Service profiles are now supported for external authorities!
+  * **New** `linkerd routes --open-api` flag that generates a service profile
+  based on an OpenAPI specification (swagger) file
+* Web UI
+  * **New** Top Routes page, served at `/routes`
+  * **New** Route metrics are now available in the resource detail pages for
+  services with configured profiles
+  * **New** You can now create and download a service profile from the Web UI
+* Controller
+  * **Improved** Controller components are now prefixed with `linkerd-` to prevent name collisions with existing resources
```
Comment (wmorgan) [inline CHANGES.md]: Differently-wrapped

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+  * **New** The `-disable-h2-upgrade` flag has been added to control automatic HTTP/2 upgrading
+* Proxy
+  * **Improved** The proxy's `tap` subsystem has been reimplemented to be more efficient and
+    and reliabile.
```
Comment (wmorgan) [inline CHANGES.md]: reliable

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
+## edge-18.12.1
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for any resource
```
Comment (siggy) [inline CHANGES.md]: this item was part of `edge-18.11.3` ;)

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+  * **New** Service profiles are now supported for external authorities!
+  * **New** `linkerd routes --open-api` flag that generates a service profile
```
Comment (wmorgan) [inline CHANGES.md]: remove "that"

File: CHANGES.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+  based on an OpenAPI specification (swagger) file
+* Web UI
+  * **New** Top Routes page, served at `/routes`
+  * **New** Route metrics are now available in the resource detail pages for
+  services with configured profiles
+  * **New** You can now create and download a service profile from the Web UI
+* Controller
+  * **Improved** Controller components are now prefixed with `linkerd-` to prevent name collisions with existing resources
+  * **New** The `-disable-h2-upgrade` flag has been added to control automatic HTTP/2 upgrading
```
Comment (siggy) [inline CHANGES.md]: maybe write this as `**New** linkerd install --disable-h2-upgrade flag...` to give context that it belongs to install.

File: CHANGES.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
+## edge-18.12.1
+
+* CLI
+  * **New** `linkerd routes` command displays per-route stats for any resource
+  * **New** Service profiles are now supported for external authorities!
+  * **New** `linkerd routes --open-api` flag that generates a service profile
+  based on an OpenAPI specification (swagger) file
+* Web UI
+  * **New** Top Routes page, served at `/routes`
```
Comment (wmorgan) [inline CHANGES.md]: capitalization

File: CHANGES.md [hunk 7]
```diff
@@ -1,3 +1,25 @@
...
+  * **New** Route metrics are now available in the resource detail pages for
+  services with configured profiles
+  * **New** You can now create and download a service profile from the Web UI
```
Comment (wmorgan) [inline CHANGES.md]: s/Web UI Controller/web UI/

File: CHANGES.md [hunk 8]
```diff
@@ -1,3 +1,25 @@
...
+* Controller
+  * **Improved** Controller components are now prefixed with `linkerd-` to prevent name collisions with existing resources
+  * **New** The `-disable-h2-upgrade` flag has been added to control automatic HTTP/2 upgrading
+* Proxy
+  * **Improved** The proxy's `tap` subsystem has been reimplemented to be more efficient and
+    and reliabile.
+    * The proxy now supports Route metadata in Tap queries and events.
+  * **Fixed** A potential HTTP/2 window starvation bug has been fixed.
+  * **Fixed** Prometheus counters now wrap properly for values greater than 2^53.
```
Comment (siggy) [inline CHANGES.md]: nit: for consistency with the previous list items, no periods at the end

File: CHANGES.md [hunk 9]
```diff
@@ -1,3 +1,25 @@
...
+  * **New** Route metrics are now available in the resource detail pages for
+  services with configured profiles
+  * **New** You can now create and download a service profile from the Web UI
+* Controller
+  * **Improved** Controller components are now prefixed with `linkerd-` to prevent name collisions with existing resources
+  * **New** The `-disable-h2-upgrade` flag has been added to control automatic HTTP/2 upgrading
+* Proxy
+  * **Improved** The proxy's `tap` subsystem has been reimplemented to be more efficient and
+    and reliabile.
+    * The proxy now supports Route metadata in Tap queries and events.
```
Comment (wmorgan) [inline CHANGES.md]: lowercase "route" and "tap"

File: CHANGES.md (hunk 8 above)
Comment (wmorgan) [inline CHANGES.md]: Would be nice to call out the contributor of this patch

File: CHANGES.md (hunk 3 above)
Comment (rmars) [reply CHANGES.md]: ooops :)

Comment (rmars) [reply CHANGES.md]: I also added some emphasis on *any resource*

**Pull request merged**